purldb-toolkit/README.rst

```diff
@@ -20,84 +25,541 @@ Use this command to get basic help::
+                },
+                "inferred_urls": [
+                    {
+                        "url": "https://www.npmjs.com/package/canonical-path/v/1.0.0",
+                        "get_request_status_code": 200,
+                        "head_request_status_code": 200
+                    },
+                    {
+                        "url": "http://registry.npmjs.org/canonical-path/-/canonical-path-1.0.0.tgz",
+                        "get_request_status_code": 200,
+                        "head_request_status_code": 301
+                    }
+                ],
+                "repository_download_url": {
+                    "url": null,
+                    "get_request_status_code": "N/A",
+                    "head_request_status_code": "N/A"
+                },
+                "repository_homepage_url": {
+                    "url": "https://www.npmjs.com/package/canonical-path/v/1.0.0",
+                    "get_request_status_code": 200,
+                    "head_request_status_code": 200
+                }
+            },
+            {
+                "purl": "pkg:rubygems/rails@7.0.0",
+                "download_url": {
+                    "url": "https://rubygems.org/downloads/rails-7.0.0.gem",
+                    "get_request_status_code": 200,
+                    "head_request_status_code": 200
+                },
+                "inferred_urls": [
+                    {
+                        "url": "https://rubygems.org/gems/rails/versions/7.0.0",
+                        "get_request_status_code": 200,
+                        "head_request_status_code": 200
+                    },
+                    {
+                        "url": "https://rubygems.org/downloads/rails-7.0.0.gem",
+                        "get_request_status_code": 200,
+                        "head_request_status_code": 200
+                    }
+                ],
+                "repository_download_url": {
+                    "url": null,
+                    "get_request_status_code": "N/A",
+                    "head_request_status_code": "N/A"
+                },
+                "repository_homepage_url": {
+                    "url": "https://rubygems.org/gems/rails/versions/7.0.0",
+                    "get_request_status_code": 200,
+                    "head_request_status_code": 200
+                }
+            }
+        ]
+    }
+
+
+Notes
+#######
+
+- None atm.
+
+
+Testing
+-------
+
+Run all purldb tests:
+
+.. code-block:: none
+
+    make test
+
+Run all purlcli non-live tests (i.e., no live network calls):
+
+.. code-block:: none
+
+    DJANGO_SETTINGS_MODULE=purldb_project.settings pytest -vvs purldb-toolkit/tests/test_purlcli.py
+
+Run all purlcli live tests (i.e., check actual API endpoints etc.)
+
+.. code-block:: none
+
+    DJANGO_SETTINGS_MODULE=purldb_project.settings pytest -vvs purldb-toolkit/tests/test_purlcli_live.py --run_live_fetch
+
 
 Funding
```

Comment:
This should ideally be moved to a separate page under root, maybe under a misc folder.

Reply:
Same for the LICENSE section below.

Reply:
@AyanSinhaMahapatra   I've created a new `Miscellaneous` folder under root containing `index.rst`, `funding.rst`, `license.rst` and `testing.rst` and moved the existing relevant text to those pages (for both PURLdb and PURLdb Toolkit).

BTW, in the RTD I see references to 

- `purldb-toolkit` (which I've been using as a placeholder name) and `PURLdb Toolkit`
- `PurlDB`, `PURLdb` and `purldb` (including `purldb is a trademark of nexB Inc.`)
- `purl` and `PURL`
- `ScanCode toolkit` and `scancode-toolkit`
- `Clearlydefined` and `ClearlyDefined`
- `Package matching`, `package matching`, `Package data`, `package data`, `package metadata`
- `MatchCode` and `Matchcode` and `matchcode`

We should probably decide on standard references.

Reply:
> I've created a new Miscellaneous folder under root containing index.rst, funding.rst, license.rst and testing.rst and moved the existing relevant text to those pages (for both PURLdb and PURLdb Toolkit).

This makes sense. 

> We should probably decide on standard references.

Yes, please open an issue in https://github.com/nexB/aboutcode/, we should have a glossary there of all the terms/names we are using and have it as a reference.

Reply:
Thanks @AyanSinhaMahapatra and done -- see https://github.com/nexB/aboutcode/issues/139